code/src/LeafletProvider/Constants/DrawingTools/Events.ts

```diff
@@ -0,0 +1,13 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace LeafletProvider.Constants.DrawingTools {
+    /**
+     * Array of strings that define the available Provider Events
+     */
+    export const Events = [
```

Comment:
Do we need this? Aren't  we using the "draw:end" (or something like that) event?